tests/foreman/cli/test_activationkey.py

```diff
@@ -951,25 +951,8 @@ def test_info_activation_keys_1(self):
 
         """
 
-    @stubbed()
-    def test_info_activation_keys_2(self):
```

Comment:
this is redundant test, info already has plenty of coverage in other tests implicitly.